core/src/main/scala/scalaz/FreeT.scala

```diff
@@ -107,6 +107,29 @@ sealed abstract class FreeT[S[_], M[_], A] {
   @deprecated("Alias for `interpret`", "7.3")
   def interpretT[T[_]](st: S ~> T)(implicit M: Functor[M]): FreeT[T, M, A] = interpret(st)
 
+  /**
+    * Runs to completion, mapping the suspension with the given transformation
+    * at each step and accumulating into the monad `M`.
```

Comment:
`Bind(Rec)` and `Applicative` give a monad, so it is not completely wrong.

I was wondering, maybe it's worth it to introduce `MonadRec` type class to `scalaz` instead of requiring `BindRec` + `Applicative`?